### ISSUE TITLE
Testing Want To Read Stats Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4644]
# Page to Test:

Want To Read - https://openlibrary.org/people/mekBot/books/want-to-read/stats

# Steps

Steps for this task are outlined in #4521

# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Sabreen-Parveen** [comment]
@bpmcneilly this page has no image, you can close this issue.
